File: Easy/Two_Sum/Two_Sum.js
     * @param {number[]} nums
     * @param {number} target
     * @return {number[]}
     */
    twoSum(nums, target) {
        let myMap = new Map();
        for (let i = 0; i < nums.length; i++) {
            if (myMap.has(target - nums[i])) {
        }
        let ans = new Array(2);
                return [myMap.get(target - nums[i]), i];
            }
            myMap.set(nums[i], i);

        return [];
    }
}

File: Easy/Two_Sum/Two_Sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
       Map<Integer, Integer> myMap = new HashMap<>();
       for (int i = 0; i < nums.length; i++) {
            int num = target - nums[i];
            if (myMap.containsKey(num)) {
                return new int[]{myMap.get(num), i};
            }
            myMap.put(nums[i], i);
       }

       return new int[2];
    }
}


